docs-ref-conceptual/TOC.yml

```diff
@@ -62,6 +62,9 @@
     - name: Install - Linux install script
       href: install-azure-cli-linux.md
       displayName: install, script, unix, bsd, linux, lfs, wsl, slackware
+  - name: Update
```

Comment:
upgrade?

Reply:
There was an `Update` section in each install page such as [Yum Update](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-yum?view=azure-cli-latest#update), should we keep consistent with it?

Reply:
Discussed offline and we agreed to use `Update` for doc consistency.